lib/service_actor/checks/must_check.rb

```diff
@@ -55,9 +55,9 @@ def initialize(input_key:, actor:, nested_checks:, value:)
 
   def check
     @nested_checks.each do |nested_check_name, nested_check_conditions|
-      check, message = define_check_and_message_from(nested_check_conditions)
+      message = prepared_message_with(nested_check_name, nested_check_conditions) # rubocop:disable Layout/LineLength
```

Comment:
I agree that `prepared` sounds a bit better here. So perhaps this can be split on several lines, instead?

```suggestion
      message = prepared_message_with(
        nested_check_name,
        nested_check_conditions,
      )
```

Reply:
In this case, will need to do this :)

```ruby
def check # rubocop:disable Metrics/MethodLength
```

Reply:
Heh. Well I think I’d rather ignore a method length cop than a line length, one :p